vertx-mongo-client/src/main/java/io/vertx/ext/mongo/package-info.java

```diff
@@ -364,6 +364,294 @@
  * {@link examples.Examples#example16_d1}
  * ----
  *
+ * == Storing/Retrieving files and binary data
+ *
+ * The client can store and retrieve files and binary data using MongoDB GridFS. The
+ * {@link io.vertx.ext.mongo.MongoGridFsClient} can be used to send a whole files or buffers
```

Comment:
From the user's perspective. Inside, it uses GridFSInputStream or GridFSOutputStream and the GridFs equivalent of a Vert.x Pump. Whole files don't go across the event bus.